### DISCUSSION TITLE
Normalise data using data factory  instead of Power BI

### DISCUSSION THREAD

**Discussion by thecloudman** [microsoft/finops-toolkit#525] [Ideas]
Power Bi is doing the bulk of the work. For large datasets, power bi takes a very long time to load the data. Is it possible to do the normalisation in data factory before Power Bi connects to the data? This way Power BI just needs to run ftx_loaddata without running through ftx_normalizedata. Not sure if that would be possible but keen to find out.

**Comment by flanakin** [comment]
We were halfway through implementing normalization in Power BI when Cost Management shipped native support for FOCUS. We shifted direction to utilize FOCUS exports instead. We just released this in [FinOps hubs 0.2](https://github.com/microsoft/finops-toolkit/releases/tag/v0.2).
